bot/code_review_bot/tasks/base.py

```diff
@@ -92,8 +96,10 @@ def clean_path(self, path):
         '''
         Helper to clean issues path from remote tasks
```

Comment:
I think we should move this stuff to the analyzer themselves, maybe let's file a Bugzilla bug to do so.
Otherwise, we need to list in WORKER_CHECKOUTS all repositories we support.

Reply:
Completely agreed.

Reply:
I filed [Bug 1564129](https://bugzilla.mozilla.org/show_bug.cgi?id=1564129) & https://github.com/mozilla/code-review/issues/45 to do that.